specification/network/resource-manager/Microsoft.Network/stable/2016-06-01/network.json

```diff
@@ -5557,9 +5557,6 @@
           "description": "path ."
         }
       },
-      "required": [
```

Comment:
I can remove the last commit, but it is broken either way. Not sure if adding it to the other PR will help, since it isn't broken in the latest version, just a few of the older ones

Reply:
PS: This was removed as "required" because the property doesn't exist on the object, which is way the validation was failing. Not sure if that would affect the generation if the property didn't exist

Reply:
I rebased out this change, but the typo-fix only PR is going to fail the build because of existing errors